### PR TITLE
Talks page

### PR DESCRIPTION
**Pull request by gnunicorn**
Adding talks page with first four talk announcements.

screen shot 2016-07-22 at 14 03 24-fullpage

### CODE REVIEW

File: _data/talks.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+\n\n
+Many said it was a mistake to start from scratch and prolong the product launch with such a new language while others praised the decision for committing to a long-term vision and leading by example.
+\n\n
+This talk will highlight obstacles and breakthroughs of one team going all in on Rust over the past year and a half including effects on development, hiring and the software itself. While all teams and software have varied needs, this presentation will offer general insights to other companies looking to use Rust in production software."
+
+
+- title: "Sensors, Servos, and Signals with Rust"
+  public: true
+  by: tim
+  desc: "Rust is a systems language ready for IoT. In this talk, we'll learn how move servos, make sensor measurements, and speak low-level communication protocols like SPI and I2C. When joined with high-level HTTP libraries, audiences will leave knowing how to create a fault-tolerant embedded device that's accessible from the cloud. Examples use the Rust language and target the Tessel microcontroller platform."
```
Comment (badboy) [inline _data/talks.yml]: `how **to** move` ?

File: _data/talks.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+- title: "Why you'd be better off using QuickCheck when porting to Rust"
+  public: true
+  by: martin
+  desc: "For about half a year I've been doing a Rust port of a CPU emulator library written in C. This talk will tell you why on earth someone would want to do that, and why the effort likely would have failed without the use of QuickCheck, which has enabled me to thoroughly compare and test that the port has identical behavior to the original (as well as the CPU spec).
+\n\n
+It will also tell the tale of macro madness, the discovery of a O(n^2) bug in the Rust compiler, and why I think most Rust tradeoffs are near-optimal, seen from someone with years of experience with C/C++, C#, Scala and Clojure. It will also explain the complications when using Cargos multi-threaded testing model interfacing a single-threaded C library, and some potential solutions."
```
Comment (badboy) [inline _data/talks.yml]: Should we make this `O(n²)`?

File: talks.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+layout: default
+permalink: /talks/
+title: Talks
+---
+
+<article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+
+  <div class="popout">
+    <section>
+        <h1>Rust 2016 Talks</h1>
```
Comment (badboy) [inline talks.html]: RustFest 2016 Talks?

File: _data/speakers.yml [hunk 4]
```diff
@@ -15,14 +17,137 @@
...
+      link: "https://twitter.com/pnkfelix"
     - icon: "github"
-      title: "hannelita"
+      title: "pnkfelix"
       link: "https://github.com/pnkfelix"
+sonja:
+  name: "Sonja Heinen"
+  nick: sonjaheinen
+  featured: true
+  desc: "Sonja is a semi-nomadic visual designer interested in the space where technology, art and society collide. In October 2014 she earned a Master‘s in Sustainable Design from Kingston University in London, then relocated to Berlin. She currently works as a Frontend developer and roams through open source communities."
```
Comment (badboy) [inline _data/speakers.yml]: Master's
Master's Degree
Masters Degree

I'm never sure what is the correct thing to use

Comment (gnunicorn) [inline _data/speakers.yml]: https://en.wikipedia.org/wiki/Master%27s_degree => _master's degree_

**Pull request merged**